Documentation: add information about existing documentation and APIs (#1009)
ansys/pydpf-core: docs/source/getting_started/compatibility.rst

version.

As new features are developed, every attempt is made to ensure backward
compatibility from the client to the server. Backward compatibility is ensured for
the 4 latest Ansys versions. For example, ansys-dpf-core module with 0.8.0 version has been
developed for Ansys 2023 R2 pre1 release, for 2023 R2 Ansys version. It is compatible with
2023 R2, 2023 R1, 2022 R2 and 2022 R1 Ansys versions.

> **It is strongly encouraged to use the latest packages available**, as far they are compatible

Reviewer: ```suggestion
**Ansys strongly encourages you to use the latest packages available**, as far they are compatible
```
